kernel/src/process_utilities.rs

```diff
@@ -161,7 +161,7 @@ pub fn load_processes_advanced<C: Chip>(
     fault_policy: &'static dyn ProcessFaultPolicy,
     require_kernel_version: bool,
     _capability: &dyn ProcessManagementCapability,
-) -> Result<(), ProcessLoadError> {
+) -> Result<usize, ProcessLoadError> {
```

Comment:
Can we return `remaining_memory` directly to keep the type information?

Reply:
You mean 'remaining_memory' instead of `unused_sram_start_addr` I defined in this function?

Reply:
Yes

Reply:
Okay! I will modify it.

Reply:
I applied this feedback